### PR TITLE
[IOTDB-3119] extend set storage group

### PR DESCRIPTION
**Pull request by SpriCoder**
See more details: https://issues.apache.org/jira/browse/IOTDB-3119
I have tested locally(3-confignode, 3-datanode):
image

### CODE REVIEW

File: antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4 [hunk 1]
```diff
@@ -75,7 +75,11 @@ syncStatement
 
 // Create Storage Group
 setStorageGroup
-    : SET STORAGE GROUP TO prefixPath
+    : SET STORAGE GROUP TO prefixPath (WITH sgAttributeClause (COMMA sgAttributeClause)*)?
+    ;
+
+sgAttributeClause
+    : (TTL | SCHEMA_REPLICATION_FACTOR | DATA_REPLICATION_FACTOR | TIME_PARTITION_INTERVAL) '=' INTEGER_LITERAL
```
Comment (MarcosZyk) [inline antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4:82]: use full name ```storageGroup```

**Review by MarcosZyk — COMMENTED** [review]
PTAL

File: antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4 (hunk 1 above)
Comment (SpriCoder) [reply antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/IoTDBSqlParser.g4]: Already fixed.

**Review by MarcosZyk — APPROVED** [review]
LGTM~

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**